lib/applications.js

```diff
@@ -17,7 +18,7 @@ const errors = require('./errors');
 //
 class Application {
   constructor(name, api, version) {
-    [this.name, this.version] = common.rsplit(name, '@');
+    [this.name, this.version] = common.rsection(name, '@');
```

Comment:
I think that it's better to update `rsplit` function to support cases from [python](https://docs.python.org/3/library/stdtypes.html#str.rsplit) where I actually took it from and name it as such.